sentry-sample/src/main/AndroidManifest.xml

```diff
@@ -30,8 +30,8 @@
 
     <meta-data android:name="io.sentry.debug" android:value="true" />
 
-    <!-- Raise a ANR event if UI thread blocked for over 2 seconds -->
-    <meta-data android:name="io.sentry.anr.timeout-interval-mills" android:value="2000" />
+    <!-- To change the time used to watch for ANR. By default it's 5 seconds (5000 below as it's in milliseconds) -->
```

Comment:
@bruno-garcia 5s the App. is gonna die and nothing is gonna happen.
I'd suggest like 4s or 4.5s

Reply:
I've changed ANR default's value to 4s